tests/test_il_regional_transit.py

```diff
@@ -52,7 +52,7 @@ def test_id():
 
 
 def test_status():
-    assert parsed_items[0]['status'] == 'confirmed'
+    assert parsed_items[0]['status'] == 'passed'
```

Comment:
I had previously written a bad test that was time dependent. Just started failing.